web/ui/two-steps-page.svelte

```diff
@@ -23,25 +27,42 @@
     })
   }
 
+  function handleEscapeKey(event: KeyboardEvent): void {
+    if (backRoute.get() && event.key === 'Escape') {
+      let { route } = router.get()
+      if (route === 'fast' || route === 'slow') {
+        event.stopPropagation()
```

Comment:
This line block `keyux` behavior, which reduce UX.

Here is use case:
1. Open `Add feed`
2. Enter `github.blog`
3. Press Enter (go to candidates list)
4. Press Enter (go to candidate sidebar)
5. Press Esc. Expected: sidebar is closed, focus is back in candidate list. Actual: sidebar is closed but no element is in the focus.

Reply:
Why do we need to disable `keyux` at all?

Reply:
I suggest using blocking `keyux` jump back only on the `fast` and `slow` routes. When the user focuses on the `Open post` button and presses `Enter`, the focus remains on the `Open post` button. Therefore, when the user closes the side panel with `Escape`, we don't need to jump back to the categories list but should keep the `Open post` button in focus.

This behavior can be seen here: [video](https://drive.google.com/file/d/1rXhIkkSQ0cTVAuubfS5Juby-cbGuxXIJ/view?usp=sharing).

However, on the `add` and `categories` routes, we still use the `keyux` jump back when `Escape` is pressed.

This behavior can be seen here: [video](https://drive.google.com/file/d/1yDopW2odQ2-RGy4KWbpyrrbNLtI11JAl/view?usp=drive_link).

Reply:
> I suggest using blocking keyux jump back only on the fast and slow routes

fast/slow just has not `keyux` integration yet, so don’t look on them.

When I will have final design for these page, I will add special `aria` and focus will jump from post card to post side panel (like in Feed Add).

(Sorry, I forgot to mention it in the task 😅)


> However, on the add and categories routes, we still use the keyux jump back when Escape is pressed.
> This behavior can be seen here: [video](https://drive.google.com/file/d/1yDopW2odQ2-RGy4KWbpyrrbNLtI11JAl/view?usp=drive_link).

This video looks great. Just double-check that everything works on mouse and in the mouse/keyboard mixed scenarios.

If yes, let’s just keep `keyux` logic and just add a logic to close sidepanel.